atlp-rwanda/vikings-ec-fe
# #184759902 implement reset-password

### What does this PR do?
This PR implements a Reset password

### Description of Tasks to be Completed?
 - Users should be able to reset the password. 

### How should this be manually tested?
1. start [Vikings backend](https://github.com/atlp-rwanda/vikings-ec-bn.git) at port 5000 with FRONTEND_URL=http://localhost:8000 in your env
2. Clone [repo](https://github.com/atlp-rwanda/vikings-ec-fe.git)
3. Change into the directory `vikings-ec-fe`
4. Checkout to the branch: `feature/implement-reset-password-184759902`
5. Run the command `npm install` to install all packages
6. Run the command `npm run start`
7. go to `/auth/forget-password`
8. fill the form then `send`
9. check your email
10. click `reset password button`


#### Screenshot

![ResetPassword](https://user-images.githubusercontent.com/60066777/231389253-dde96074-a067-4b78-af00-817932b56064.png)
![FogetPassword](https://user-images.githubusercontent.com/60066777/231389362-76d7bf0a-8563-473f-bad8-3bd898221b8c.png)


### Figma

![ResetPasswordF](https://user-images.githubusercontent.com/60066777/231389986-4f350d6e-9d31-4586-9d4c-0771784e6efa.png)
![ForgetPasswordFig](https://user-images.githubusercontent.com/60066777/231390784-d4d8b87a-fa62-42a1-83ab-b436099ae34a.png)


### What are the relevant pivotal tracker stories?
[#184759902](https://www.pivotaltracker.com/story/show/184759902)
